### ISSUE TITLE
RTK Query: How to check state of query without triggering it?

### ISSUE THREAD

**Issue by ketain** [reduxjs/redux-toolkit#1652]
### What I want to accomplish
It's rather simple really. 
A child dialog component sends a query and is then closed. 
The parent component should then display a loading indicator. 

### The Question
How do I inform the parent about the state of a query it did not sent itself?

**Comment by jjl-genemod** [comment]
You should be able to do the following if there is only one cached result:

```
const { data, isError, isFetching } = useMyQuery(undefined as any);
```

Read more here:
https://redux-toolkit.js.org/rtk-query/usage/queries#query-cache-keys

**Comment by phryneas** [comment]
@jjl-genemod that would trigger it.

@ketain if you really want to see the state of a **query** without triggering it, you can use [api.endpoints.yourEndpoint.useQueryState](https://redux-toolkit.js.org/rtk-query/api/created-api/hooks#usequerystate).

But I assume, you want to know about the state of a **mutation**, since for a query you would usually not care about triggering it or not (queries can be re-triggered by RTKQ on various occasions).

In the current stable release, it is not possible to share mutation state over components, but we yesterday released a beta release of RTK 1.7 which will come with that functionality (`fixedCacheKey`).

You can [find that release here](https://github.com/reduxjs/redux-toolkit/releases/tag/v1.7.0-beta.0)

As for how to use it, see [the beta documentation](https://deploy-preview-1565--redux-starter-kit-docs.netlify.app/rtk-query/usage/mutations#shared-mutation-results)
